docs/devs/connect-with-band.md

```diff
@@ -7,7 +7,9 @@ Now that you are onboard with us. Let's get started on to the actual implementat
 If you are building a new project, by using [Truffle](https://www.trufflesuite.com/), you can bootstrap a new Ethereum-based project from a prepared boilerplate. Run the command below to unbox a new Band-powered project into your current directory.
```

Comment:
Change to

Run the commands below to create a new project under a new folder named `my_project` with Band Protocol connection pre-installed.